nix/cells/automation/library/plutus-benchmark-runner.sh

```diff
@@ -29,40 +24,47 @@ if [ -z "$PR_NUMBER" ] ; then
    echo "[ci-plutus-benchmark]: 'PR_NUMBER' is not set! Exiting"
    exit 1
 fi
+if [ -z "$PR_COMMIT_SHA" ] ; then
+   echo "[ci-plutus-benchmark]: 'PR_COMMIT_SHA' is not set! Exiting"
+   exit 1
+fi
 if [ -z "$BENCHMARK_NAME" ] ; then
    echo "[ci-plutus-benchmark]: 'BENCHMARK_NAME' is not set! Exiting"
    exit 1
 fi
+if [ -z "$GITHUB_TOKEN" ] ; then
+   echo "[ci-plutus-benchmark]: 'GITHUB_TOKEN' is not set! Exiting"
+   exit 1
+fi
+
+echo "[ci-plutus-benchmark]: Cloning the plutus repository"
```

Comment:
Do we need to do this? This makes the script much harder to use locally. Before the checkout happened outside the test runner, can we not arrange for that to happen in the same way that we do for normal CI runs?

Reply:
IIRC when I discussed this with @pacman I learned that in cicero we start with an empty directory and so we have to checkout the repo ourselves each time. Hence why I added those git commands to the script. 
If there is a way to workaround that we could probably shave off some time? 
As for running locally, we can indeed remove that check and maybe echo a warning

Reply:
Just removed it, tullia can do this for us with the git preset